data/mods/Xedra_Evolved/items/comestibles/goblin_fruits.json

```diff
@@ -0,0 +1,313 @@
+[
+  {
+    "type": "COMESTIBLE",
+    "id": "goblin_fruit_abstract",
+    "name": { "str": "Abstract Goblin Fruit" },
+    "description": "Somehow you got a hold of the platonic goblin fruit, the one fruit to rule them all.  It's a bug if you have this, but the lords of the Bright Lands will surely pay a high price for it.",
```

Comment:
I chuckled at this line.  Also just an FYI, XE has added some additional fruit bearing trees at some spillage locations.  Figs, pomegranates and Hawthorns.  In case you were looking for other goblin fruit types.

Reply:
I'll take a look!  I do think though that I'm going to go through and make the goblin fruits not tied to existing Earthly fruits--like the "sweet goblin fruit" I worked on before I had to call it a night last night, I want them to be weird and not automatically identifiable as a mundane fruit.  Cloying goblin fruit, translucent goblin fruit, bumpy goblin fruit, acrid goblin fruit, etc.

Reply:
Very cool!

Reply:
I just uploaded some goblin fruit work so you can see the direction I'm taking it!